src/common/vitals/vital-metric.js

```diff
@@ -9,7 +9,7 @@ export class VitalMetric {
   }
 
   update ({ value, attrs = {} }) {
-    if (value < 0) return
+    if (value === undefined || value === null || value < 0) return
```

Comment:
Could just check if `value` is type of `number`
```suggestion
    if (typeof value !== 'number' || value < 0) return
```